packages/contracts/contracts/Tally.sol

```diff
@@ -113,6 +105,23 @@ contract Tally is Clone, SnarkCommon, Hasher, DomainObjs, ITally {
     }
   }
 
+  /// @notice Get the tally results for a specific index
+  /// @param index The index to query
+  /// @return The tally result
```

Comment:
If the code is moved to the interface, you should inherit docs from it to prevent duplication.